app/src/main/java/com/infomaniak/mail/ui/MainViewModel.kt

```diff
@@ -196,6 +199,24 @@ class MainViewModel @Inject constructor(
     }
     //endregion
 
+    //region Current Thread
+    val currentThreadUid = MutableLiveData<String?>(null)
+
+    val isInThread get() = currentThreadUid.value != null
```

Comment:
`isInThread` is unused in this PR, is it normal?

Reply:
Oversight, it's fixed.